Makefile

```diff
@@ -33,12 +34,14 @@ endif
 
 MSBUILD_CMD = $(ROSLYN_TOOLSET_PATH)/corerun $(ROSLYN_TOOLSET_PATH)/MSBuild.exe $(MSBUILD_ARGS)
 
-all: toolset
+.PHONY: all bootstrap test restore toolset
```

Comment:
It marks these targets as things that are manually run, not files to be depended on (otherwise the left hand side of a make target is a file, whose timestamp is checked for freshness).